wordpress-mobile/WordPress-Android
# Stories in the updated my site

Fixes #13531 

This PR is copying some of the Stories logic from MySiteFragment into ImprovedMySiteFragment. I've extracted some parts but unfortunately the code is not really testable (because of how the stories code is written - there are static objects, it uses Intent and the Android classes don't work in unit tests). 

To test:
- Turn on the flag and restart the app
- Click on the FAB on My Site and on "Add Story"
- Create a story with items from your device
- Notice you can finish the story flow and create a story
- Notice the snackbars are visible
- Repeat the flow with media from WP Media library

PR submission checklist:
